superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/plugin/controlPanel.ts

```diff
@@ -21,7 +21,7 @@ import {
   validateNonEmpty, FeatureFlag, isFeatureEnabled,
   QueryMode,
   QueryFormColumn,
-  ChartDataResponseResult,
+//  ChartDataResponseResult,
```

Comment:
I would remove this,. If we un-comment the code below at some point, we can add this line again.

Reply:
That makes sense to me.  I've pushed a change to remove the line and fix some punctuation in one of my comments to make it more readable.